app/tests_integration.py

```diff
@@ -93,3 +93,27 @@ def test_edit_user_with_valid_data(self):
         self.assertEqual(editedClient.phone, client.phone)
         self.assertEqual(editedClient.address, client.address)
         self.assertEqual(editedClient.email, client.email)
+
+class ProductTest(TestCase):
+    def test_zero_price_validation(self):
+        response = self.client.post(
+            reverse("products_form"),
+            data={
+                "name": "Pelota",
+                "type": "Juguete",
+                "price": str(0),
```

Comment:
could have used "0" instead but it's ok

Reply:
porque me comentas en ingles si sabes que hablamos los dos español, queres que lo cambie?

Reply:
no es necesario pero para tener en cuenta para la proxima 👍